plugin/pkg/admission/imagepolicy/config_test.go

```diff
@@ -37,8 +37,8 @@ func TestConfigNormalization(t *testing.T) {
 				RetryBackoff: ((minRetryBackoff + maxRetryBackoff) / 2) / time.Millisecond,
 			},
 			normalizedConfig: imagePolicyWebhookConfig{
-				AllowTTL:     ((minAllowTTL + maxAllowTTL) / 2) / time.Second * time.Second,
-				DenyTTL:      ((minDenyTTL + maxDenyTTL) / 2) / time.Second * time.Second,
+				AllowTTL:     ((minAllowTTL + maxAllowTTL) / 2) / time.Second,
+				DenyTTL:      ((minDenyTTL + maxDenyTTL) / 2) / time.Second,
```

Comment:
The original was in fact correct.

`(minAllowTTL + maxAllowTTL) / 2) / time.Second` is the original, non-normalized value. The multiplication with `time.Second` is the expected normalization.

Fun fact:
One would think "I can just omit the division by `time.Second` and I don't have to use the multiplication anymore". However, `((minDenyTTL + maxDenyTTL) / 2)` yields a number that, when translated to seconds, has a decimal part. The division by `time.Second` converts it to whole seconds and this decimal part is discarded.

That leads to
```
((minAllowTTL + maxAllowTTL) / 2) != ((minAllowTTL + maxAllowTTL) / 2) / time.Second * time.Second
```
which would make the test fail.